<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1" colspan="2">
     <widget class="QTextBrowser" name="outputbox">
      <property name="styleSheet">
       <string notr="true">QTextBrowser{
	border: 0px solid ;border-radius:8px; background-color: palette(base); 
}

QTextBrowser:hover{
	border: 1px solid rgb(193,225,193); border-radius:8px; background-color : palette(base)
}
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="3">
     <widget class="QPushButton" name="Submitbutton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="binaryadditionbutton">
      <property name="text">
       <string>Binary Addition</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="userentry">
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
	border: 0px solid ;border-radius:8px; background-color: palette(base); 
}

QTextEdit:focus{
	border: 1px solid rgb(255,105,97); border-radius:8px; background-color : palette(base)
}

QTextEdit:hover{
	border: 1px solid rgb(255,105,97); border-radius:8px; background-color : palette(base)
}</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <widget class="QComboBox" name="choice2">
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Binary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Denary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="choice1">
      <property name="styleSheet">
       <string notr="true">QComboBox{

}</string>
      </property>
      <item>
       <property name="text">
        <string>Select</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Binary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Denary</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>SF Pro</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>Converter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>to</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="tryagain.qrc"/>
 </resources>
 <connections/>
</ui>
